zk_prover/src/chips/range/tests.rs

```diff
@@ -103,6 +103,32 @@ struct TestCircuit<const N_BYTES: usize> {
     pub b: Fp,
 }
 
+impl<const N_BYTES: usize> TestCircuit<N_BYTES> {
```

Comment:
My fault, haven't noticed this while reviewing the previous PR. Since there's actually some common code shared between different circuits, it can stay in CircuitBase 😄